webserver/templates/data/data.html

```diff
@@ -3,69 +3,18 @@
 {%- block content -%}
   <h2 class="page-title">AcousticBrainz data</h2>
   <p><ul>
-      <li><a href="#getting-data">Getting data</a>
-      <li><a href="#highlevel-data">Highlevel data and datasets</a>
+      <li><a href="#api-reference">API-Reference</a>
       <li><a href="#sample-data">Sample data</a>
   </ul></p>
   <p>
     Before diving into the API details, please have a look at the
     <a href="#sample-data">sample data</a> below.
   </p>
 
-  <a name="getting-data"></a>
-  <h3>Getting data</h3>
+  <a name="api-reference"></a>
+  <h3>API-Refernce</h3>
```

Comment:
There is a typo here and the hyphen isn't needed in the heading.
```suggestion
  <h3>API Reference</h3>
```